advanced/kernel-messaging/README.md

```diff
@@ -10,85 +10,89 @@
 ## Custom Kernel Interactions: Kernel Managment and Messaging
 
 One of the main features of JupyterLab is the possibility to manage and
-interact underlying compute kernels. In this section, we explore how to
-start a kernel and execute a simple command on it.
+interact with execution [kernels](https://jupyter-client.readthedocs.io/en/latest/kernels.html). In this example, youu will explore how to
+start a kernel and execute a command on it.
```

Comment:
In this case it's more about sending some code to execute?

Reply:
You are right. I rephrased that one.